modules/farm_fd2_school/src/entrypoints/FD2_Application/App.vue

```diff
@@ -182,21 +182,30 @@ export default {
         this.unit.attributes.name
       );
       console.log(quantity);
+      const plantAssets = await farmosUtil.getPlantAsset(this.pickedPlant.uuid);
+      await farmosUtil.createHarvestLog(
+        this.date,
+        this.pickedPlant.location,
+        this.pickedPlant.bed,
+        plantAssets,
+        quantity
```

Comment:
- The comment should also be submitted as part of the harvest log.  See the documentation for `createHarvestLog` for where the argument for the comment goes.